server/api/documents.js

```diff
@@ -332,7 +331,7 @@ router.post('documents.create', auth(), async ctx => {
 });
 
 router.post('documents.update', auth(), async ctx => {
-  const { id, title, text, publish, lastRevision } = ctx.body;
+  const { id, title, text, publish, done, lastRevision } = ctx.body;
```

Comment:
`done` is passed if the user has finished editing the document, which triggers the `documents.update` event. Open to improved naming on this one.